examples/ray_tracing_triangle_meshes/source/ray_tracing_triangle_meshes.cpp

```diff
@@ -211,13 +211,16 @@ class ray_tracing_triangle_meshes_app : public gvk::invokee
 
 		mPipeline.enable_shared_ownership(); // Make it usable with the updater
 		mUpdater.emplace();
+		
+		// As we are updating our image views if swap chain is resized, it is advised to also cleanup our descriptor cache
+		std::function<void(void)> descriptorCleanup = [this]() {this->mDescriptorCache.cleanup(); };
```

Comment:
For this PR, we can leave it like that. However, it is not an optimal solution. There should be more fine-grained control over descriptor cache cleanup (but that's beyond the scope of this PR).

Reply:
see PM